docs/current_docs/integrations/838502-php.mdx

```diff
@@ -0,0 +1,95 @@
+---
+slug: /integrations/838502/php
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# PHP
+
+Dagger can be used to perform common CI tasks - testing, containerizing, publishing and more - for any PHP application.
+
+The following code sample demonstrates how these CI tasks can be encapsulated as Dagger Functions in a Dagger module. It assumes:
+
+- A PHP 8.2.x Web application with:
+  - Composer for package management
+  - PHPUnit for application testing
+- Credentials to publish the containerized application image to a registry
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+Bootstrap a new module:
+
+```shell
+dagger init --name=my-module --sdk=go
+```
+
+Update the generated `dagger/main.go` file with the following code:
+
+```go file=./snippets/php/go/main.go
+```
+
+</TabItem>
+<TabItem value="Python">
+
+Bootstrap a new module:
+
+```shell
+dagger init --name=my-module --sdk=python
+```
+
+Update the generated `dagger/src/main.py` file with the following code:
+
+```python file=./snippets/php/python/main.py
+```
+
+</TabItem>
+<TabItem value="TypeScript">
+
+Bootstrap a new module:
+
+```shell
+dagger init --name=my-module --sdk=typescript
+```
+
+Update the generated `dagger/src/index.ts` file with the following code:
+
+```typescript file=./snippets/php/typescript/index.ts
+```
+
+</TabItem>
+</Tabs>
+
+:::warning
+The code sample above is illustrative only. Modify it to your application's specific requirements.
+:::
+
+Here is an example of calling the `Test()` Dagger Function to run an application's unit tests:
```

Comment:
This and `Publish()` below are using Go's conventions. Maybe use the CLI's kebab convention instead, which works in this context of `dagger call`?

Reply:
Done